polychrom/simulation.py

```diff
@@ -89,24 +89,28 @@
 
 
 from collections.abc import Iterable
-
-import simtk.openmm as openmm
+try:
+    import openmm
+except Exception:
+    import simtk.openmm as openmm
 import simtk.unit
 
-from . import forces
+from polychrom import forces
+
 
 logging.basicConfig(level=logging.INFO)
 
 # updated manually every now and then
-VER_LATEST = "7.5"
-VER_DATE = "2020-12-15"
+VER_LATEST = "7.7"
+VER_DATE = "2022-03-13"
 
-ver_cur = openmm.__version__
-if ver_cur < VER_LATEST:
-    warnings.warn(f"\n WARNING: you have OpenMM {ver_cur}; {VER_LATEST} is the latest as of {VER_DATE}, "
+if hasattr(openmm, "__version__"):
+    ver_cur = openmm.__version__
+    if ver_cur < VER_LATEST:
+        warnings.warn(f"\n WARNING: you have OpenMM {ver_cur}; {VER_LATEST} is the latest as of {VER_DATE}, "
                   "Upgrade is recommended.")
-    print("to upgrade openmm, run --->  conda install -c conda-forge openmm")
-    print("Ideally in a new conda environment")
+        print("to upgrade openmm, run --->  conda install -c conda-forge openmm")
+        print("Ideally in a new conda environment")
```

Comment:
This part was tricky sometimes, because OpenMM imported in some way worked, but lacked __version__ attribute 😱